mrtt-ui/src/components/CausesOfDeclineForm.js

```diff
@@ -456,6 +456,11 @@ function CausesOfDeclineForm() {
                 })}
               </Box>
             ))}
+            <ErrorText>
+              {errors.causesOfDecline?.length
```

Comment:
a bit of a hack here. We know that if there are any errors, it has to do with missing levels of degradation items. It's better than filtering through every possible nested item to figure out which drop down is missing a selection

Reply:
if its a hack, maybe it needs a comment explainer in code?
Or 
```
const descriptiveExplainingNameForHAck = errors.causesOfDecline?.length

...
  <ErrorText>
              { descriptiveExplainingNameForHAck ? ......
              
```